src/common_interface/algorithms.jl

```diff
@@ -73,7 +91,24 @@ Base.@pure function CVODE_BDF(;
         prec_side,
     )
 end
+"""
+CVODE_Adams: CVode Adams-Moulton solver.
+
+method - This is the method for solving the implicit equation. For BDF this defaults to :Newton while for Adams this defaults to :Functional. These choices match the recommended pairing in the Sundials.jl manual. However, note that using the :Newton method may take less iterations but requires more memory than the :Function iteration approach.
+linear_solver - This is the linear solver which is used in the :Newton method.
+The choices for the linear solver are:
 
+:Dense - A dense linear solver.
```

Comment:
any reason why the docs on the other args are cutoff?

Reply:
Good catch, the documentation was a little bit non-linear and I missed it xD